### ISSUE TITLE
Add Federated ETL Pipeline

### ISSUE THREAD

**Issue by kaelanspatel** [kubecost/cost-analyzer-helm-chart#1700]
Current federation of multiple clusters relies on Thanos and building ETL data from combined metrics. The goal of adding a federated ETL is to mitigate memory and efficiency issues with the Thanos implementation, including costly builds and shipment of unnecessary unaggregated metrics data. 

This would be done by creating a pipeline that uses existing integrations with s3 to federate clusters' ETL data, designating an install of Kubecost to act as a "Federator" which consolidates the data.

┆Issue is synchronized with this [Jira Task](https://kubecost.atlassian.net/browse/GIB-178) by [Unito](https://www.unito.io)

**Comment by dwbrown2** [comment]
This actually completed? Seems like a great tracking issue for Federated ETL work in 1.98!

**Comment by Adam-Stack-PM** [comment]
@dwbrown2 This part of the work is completed! We could have been more clear in what this issue covered. 

This issue relates only to the initial beta of Federated ETL upload. This will designate an s3 bucket which will be used to upload ETL data from federated clusters to be later used for Federated ETL.

@kaelanspatel and I went back and forth about including this in the release notes. 

We decided not to include it to avoid confusion as this part of the work does not add value without the other federated work planned for v1.98. At the same time, we would like to reach out to a few customers to get feedback on the upload beta work.
